proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+
+**Examples**
+1. New projects that are designed to extend one or more _$FOUNDATION_ projects with functionality or interoperability libraries. 
+1. Independent projects that fit the _$FOUNDATION_ mission and provide potential for a novel approach to existing functional areas (or are an attempt to meet an unfulfilled need).
+1. Projects commissioned or sanctioned by the _$FOUNDATION_.
+1. Any project that realistically intends to join the _$FOUNDATION_ Incubation or Top Level Stages in the future and wishes to lay the foundations for that transition.
+
+**Expectations**
+End users should evaluate Sandbox projects with care - some Sandbox projects may fail, and others may stay in the  Sandbox Stage indefinitely. Sandbox stage projects will receive minimal marketing support from the _$FOUNDATION_. Sandbox Stage projects will be reviewed on an annual basis; they may also request a status review by submitting a report to the CPC.
+
+**Acceptance Criteria**
+To be considered for Sandbox Stage, the project must meet the following requirements:
+* 2 CPC sponsors to champion the project & provide mentorship
+* A presentation to at the meeting of the CPC
+* Adherence to the _$FOUNDATION_ IP Policy
+* Upon acceptance, Sandbox projects must list their status prominently on website/readme
+
+### Incubating Stage*
+**Definition** 
+The Incubation Stage is for projects that are interested in reaching the Top-Level Stage, or want to have more access, participation, responsibilities and representation in the CPC. Incubating projects will receive mentorship from the CPC and are expected to actively develop their community of contributors, governance, and other project documentation. 
+
+**Examples**
+1. Projects that are on their way or very likely to become Top Level Projects.
+
+**Expectations**
+Projects can remain in the Incubating stage indefinitely, but they are generally expected to graduate within two years.
+
+**Acceptance Criteria**
+To be considered for Incubation Stage, the project must meet the Sandbox requirements as well as the following:
+ * Document that it is being used successfully in production by at least three independent end users which, in the CPC’s judgement, are of adequate quality and scope.
+ * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+ * Demonstrate a substantial ongoing flow of commits and merged contributions.
+ * Since these metrics can vary significantly depending on the type, scope and size of a project, the CPC has final judgement over the level of activity that is adequate to meet these criteria.
+ * Recieve a two-thirds supermajority vote of the CPC to move to Incubation Stage. 
```

Comment:
I think a project in Incubating stage _must_ have some form of documented governance. A BFDL is fine at this stage.

Reply:
I'm not sure BFDL should be a good option, maybe we need to set an expectation of more open governance?  Not sure ....

Reply:
The key thing in my mind is that if the project is going to get funding from the members, there should be a transparent and fair way for those members to have people engage and particpate in the project (not because they paid to be part of the foundation, but because like everybody else they contribute to the project).

Reply:
I now see the discusion later on.  I think if the requirement for a top level project is to be Open Governance, they maybe the minimal requirement for incubation is to be "on the path to open governance".

Reply:
Minor nit
```suggestion
 * Receive a two-thirds supermajority vote of the CPC to move to Incubation Stage. 
```

Reply:
good catch :D

Reply:
thank you!